App/HejtoBuzz/HejtoBuzz.xcodeproj/project.pbxproj

```diff
@@ -106,6 +109,7 @@
 		640ECDBB29B3F4AE00D7F131 /* Packages */ = {
 			isa = PBXGroup;
 			children = (
+				696AA50129BCD8050054478B /* HejDTO */,
```

Comment:
Could you right-click on the `Packages` and sort them by type? :)

Reply:
Fixed :)